extensions/amp-analytics/0.1/amp-analytics.js

```diff
@@ -224,14 +230,14 @@ export class AmpAnalytics extends AMP.BaseElement {
    * Returns a promise that resolves when remote config is ready (or
    * immediately if no remote config is specified.)
    * @private
-   * @return {!Promise<undefined|JSONType>}
+   * @return {!Promise<undefined>}
```

Comment:
It is helpful. It say: "This promise produces nothing". Without the `undefined` it might produce `?`